src/sample/conference/src/main/java/owt/sample/conference/VideoFragment.java

```diff
@@ -151,14 +156,25 @@ void updateStats(RTCStatsReport report, boolean outbound) {
                         lastBytesReceived = bytes;
                     }
 
-                    packetsSR = (long) members.get(outbound ? "packetsSent" : "packetsReceived");
+                    long currentFrame = (long) members.get(outbound ? "framesEncoded" : "framesDecoded");
+                    if (outbound) {
+                        frameRate = (currentFrame - lastFrameEncoded) * 1000
```

Comment:
outbound/inbound check should be done after you calculate the frameRate. Reduce un-neccessary redundant code.

Reply:
done